### DISCUSSION TITLE
BTRFS Raid Support

### DISCUSSION THREAD

**Discussion by inyourface09** [cockpit-project/cockpit#21049] [Ideas]
![Screenshot 2024-09-28 195556](https://github.com/user-attachments/assets/8da6bcd3-d4d8-4e4b-9772-1c0aac92ad48)
Drives in attached picture are zoned 4 drive mirror and mounted. Storaged is unable to show the drives correctly and or utilize BTRFS. 
Anything I can do to help make this request a reality, please ping me.

**Comment by jelly** [comment]
Can you show the mount output? And maybe also the warning you get when you click on `sdg`?

**Reply by inyourface09** [reply]
The World UUID was manually mount via fstab, storage still showed the same error before manual mounting. lsblk seems unable to see the raid structure.

**Comment by inyourface09** [comment]
![Mount point](https://github.com/user-attachments/assets/7e2af04c-6234-41d9-931d-f013fd94a179)
![btrfs filesystem show output](https://github.com/user-attachments/assets/19aabd57-cc14-41bc-a721-62ae70377fa0)
![lsblk output](https://github.com/user-attachments/assets/d61ffb9b-0402-4ac6-a076-6377dca9b5a3)

**Reply by jelly** [reply]
How does fstab look like?
